src/components/GamePage.vue

```diff
@@ -0,0 +1,672 @@
+    p.value.querySelectorAll(".slash")[0].classList.remove("bg-indigo-500");
+  }
+  else if(event.key === "_"){
+    p.value.querySelectorAll(".sub")[0].classList.remove("bg-indigo-500")
+  }
+  else if(event.key === " "){
+    p.value.querySelectorAll(".space")[0].classList.remove("bg-indigo-500")
+  }
+  else if(p.value.querySelectorAll("." + event.key).length === 0){
+    console.log("none")
+  }
+  else{
+    p.value.querySelectorAll("." + event.key)[0].classList.remove("bg-indigo-500")
+  }
+};
+
+//中身をrefresh
+const refreshes = () => {
+  inputs.value = ""
+}
+
+</script>
+
+<template class="box">
+  <div>
+    <!-- 上半分のHTML -->
+    <div class="upperbox mt-2 mb-2 bg-white flex justify-around items-center">
+      <div class="codearea flex justify-center  items-center ">
+        <textarea
+          cols="55" rows="55"
+          class="codearea p-5"
+          type="textarea"
+          placeholder="
+          <script setup lang='ts'>
+          import { RouterLink, RouterView } from 'vue-router'
+          import HeaderAll from '@/components/HeaderAll.vue';
+          </script>
+          
+          <template>
+            <HeaderAll />
+            <div class=7box bg-yellow-400'>
+              <div class='upperbox bg-white flex justify-center items-center'>
+                <div class='codearea flex justify-center items-center '>
+                  <input class='codearea p-5' type='textarea' placeholder='hello' disabled>
+                </div>
+              </div>
+            </div>
+          </template>
+          <script setup lang='ts'>
+          import { RouterLink, RouterView } from 'vue-router'
+          import HeaderAll from '@/components/HeaderAll.vue';
+          </script>
+          
+          <template>
+            <HeaderAll />
+            <div class=7box bg-yellow-400'>
+              <div class='upperbox bg-white flex justify-center items-center'>
+                <div class='codearea flex justify-center items-center '>
+                  <input class='codearea p-5' type='textarea' placeholder='hello' disabled>
+                </div>
+              </div>
+            </div>
+          </template>
+          <script setup lang='ts'>
+          import { RouterLink, RouterView } from 'vue-router'
+          import HeaderAll from '@/components/HeaderAll.vue';
+          </script>
+          
+          <template>
+            <HeaderAll />
+            <div class=7box bg-yellow-400'>
+              <div class='upperbox bg-white flex justify-center items-center'>
+                <div class='codearea flex justify-center items-center '>
+                  <input class='codearea p-5' type='textarea' placeholder='hello' disabled>
+                </div>
+              </div>
+            </div>
+          </template>
+          "
+          disabled></textarea>
+      </div>
+
+      <div class="codearea flex-col justify-center  items-center">
+        <textarea
+          v-on:keydown="KeyDown()"
```

Comment:
```suggestion
          @keydown="KeyDown()"
```
[SHOULD]
ここのコーディング規約の話になりますが、省略記法があるのでそちらに合わせたいです。
https://jp.vuejs.org/v2/guide/syntax.html#:~:text=%3C!%2D%2D%20%E7%9C%81%E7%95%A5%E8%A8%98%E6%B3%95%20%2D%2D%3E%0A%3Ca%20%40click%3D%22doSomething%22%3E%20...%20%3C/a%3E